components/uKasd89kldsIK0Pd.yml

```diff
@@ -0,0 +1,23 @@
+name: bot-emotion-classifier
```

Comment:
same

Reply:
btw why bot emotion cls is annotator not bot uttr annotator?

Reply:
name fixed
don't know why I didn't fixed type of the annotator before)
now should be correct: changed state manager method and formatters